src/main/java/az/edu/strangers/Pet.java

```diff
@@ -68,17 +71,17 @@ public void setTrickLevel(Integer trickLevel) {
         this.trickLevel = trickLevel;
     }
 
-    public String[] getHabits() {
+    public Set<String> getHabits() {
         return habits;
     }
 
-    public void setHabits(String[] habits) {
+    public void setHabits(Set<String> habits) {
         this.habits = habits;
     }
 
     public String toString() {
         return "%s{nickname='%s', age=%d, trickLevel=%d, habits=%s}"
-                .formatted(species, nickName, age, trickLevel, Arrays.toString(habits));
+                .formatted(species, nickName, age, trickLevel, habits);
```

Comment:
does habits work? I think getHabits is more suitable

Reply:
habits is in Pet class. why bother call getHabits when you can directly access habits?

Reply:
it's just a habit:))